x/bridge/types/keys.go

```diff
@@ -14,10 +16,26 @@ const (
 	QuerierRoute = ModuleName
 )
 
-var InboundTransfersKey = []byte{0x01}
+var (
+	InboundTransfersKey   = []byte{0x01}
+	FinalizedTransfersKey = []byte{0x02}
+	LastHeightsKey        = []byte{0x03}
+)
 
 // InboundTransferKey returns the store prefix key where all the data
 // associated with a specific InboundTransfer is stored
-func InboundTransferKey(externalID string) []byte {
-	return append(InboundTransfersKey, []byte(externalID)...)
+func InboundTransferKey(externalID string, externalHeight uint64) []byte {
+	return binary.BigEndian.AppendUint64(append(InboundTransfersKey, []byte(externalID)...), externalHeight)
+}
```

Comment:
I think for safety reasons we need to use a key separator, given the externalID can really take any shape. Its likely a non issue but safety here is pretty paramount, wdyt?

Reply:
Done